content/en/docs/concepts/architecture/nodes.md

```diff
@@ -14,7 +14,12 @@ A node may be a virtual or physical machine, depending on the cluster. Each node
 is managed by the
 {{< glossary_tooltip text="control plane" term_id="control-plane" >}}
 and contains the services necessary to run
-{{< glossary_tooltip text="Pods" term_id="pod" >}}.
+{{< glossary_tooltip text="Pods" term_id="pod" >}}. A Node object is an object in 
+the Kubernetes API that represents a node.
+
+It is important to note the difference between a Node and a Node object. A node is a 
+physical or virtual machine that runs applications or workloads, while
+a node-object is a representation of that node in the Kubernetes API.
```

Comment:
This text is confusing because you are not following our style guidelines.
Whenever we talk about a resource kind, we use the original spelling which is usually in CamelCase, e.g. Node, Pod. When we are talking about a generic concept, such as a k8s cluster member running application, we call it a node. We say Container when we refer to the Container object defined by the API, and we say container when the concept is generic.